satMutMPRA.pm

```diff
@@ -0,0 +1,274 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <https://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl helpdesk at
+<https://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+satMutMPRA - saturation mutagenesis-based massively parallel reporter assays (satMutMPRA) for 21 regulatory elements (11 enhancers, 10 promoters) - Add satMutMPRA data fields to the VEP output
+
+=head1 SYNOPSIS
+
+ mv satMutMPRA.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin satMutMPRA,file=/path/to/satMutMPRA_data.gz,cols=col1:col2
+
+=head1 DESCRIPTION
+
+A VEP plugin that retrieves data for variants from a tabix-indexed satMutMPRA file (1-based file).
+
+Parameters can be set using a key=value system:
+ file           : required - a tabix indexed file of the satMutMPRA data corresponding to desired assembly.
+
+ pvalue         : p-value threshold  (default: 0.00001)
+
+ cols           : colon delimited list of additional data types to be returned from the satMutMPRA data
+                (only 'Value', 'P-Value', and 'Element' reported by default)
+
+satMutMPRA data was obtained for 20 disease-associated regulatory elements and one ultraconserved enhancer in different cell lines:
+ - ten promoters (of TERT, LDLR, HBB, HBG, HNF4A, MSMB, PKLR, F9, FOXE1 and GP1BB) and
+ - ten enhancers (of SORT1, ZRS, BCL11A, IRF4, IRF6, MYC (2x), RET, TCF7L2 and ZFAND3) and
+ - one ultraconserved enhancer (UC88).
+
+Please refer to the satMutMPRA web server and biorxiv manuscript for more information:
+https://mpra.gs.washington.edu/satMutMPRA/
+https://www.biorxiv.org/content/10.1101/505362v1.full
+
+The Bio::DB::HTS perl library or tabix utility must be installed in your path
+to use this plugin. The satMutMPRA data file can be downloaded from
+https://mpra.gs.washington.edu/satMutMPRA/
+
+satMutMPRA data can be downloaded for both GRCh38 and GRCh37 from the web server (https://mpra.gs.washington.edu/satMutMPRA/):
+'Download' section, select 'GRCh37' or 'GRCh38' for 'Genome release' and 'Download All Elements'.
+
+The file must be processed and indexed by tabix before use by this plugin.
+
+# GRCh38
+ > (grep ^Chr GRCh38_ALL.tsv; grep -v ^Chr GRCh38_ALL.tsv | sort -k1,1 -k2,2n ) | bgzip > satMutMPRA_GRCh38_ALL.gz
+ > tabix -s 1 -b 2 -e 2 -c C satMutMPRA_GRCh38_ALL.gz
+
+# GRCh37
+ > (grep ^Chr GRCh37_ALL.tsv; grep -v ^Chr GRCh37_ALL.tsv | sort -k1,1 -k2,2n ) | bgzip > satMutMPRA_GRCh37_ALL.gz
+ > tabix -s 1 -b 2 -e 2 -c C satMutMPRA_GRCh37_ALL.gz
+
+Note that in the last command we tell tabix that the header line starts with "Chr";
+this may change to the default of "#" in future versions of satMutMPRA.
```

Comment:
Do we know this?

Reply:
No, it was an assumption. Have removed it.